pkg/systemd/services/services.jsx

```diff
@@ -731,7 +731,14 @@ class ServicesPageBody extends React.Component {
             if (unit_path === undefined) {
                 return <EmptyStatePanel
                             icon={ExclamationCircleIcon}
-                            title={_("Unit not found")} />;
+                            title={_("Unit not found")}
+                            paragraph={
+                                <Button variant="link"
+                                        component="a"
+                                        onClick={() => cockpit.jump("/system/services", cockpit.transport.host)}>
```

Comment:
I just realized, that when you would see this for user unit, you should go to all user units and not to system ones